src/utils/errorMessage/homeworkErrorMessage.ts

```diff
@@ -0,0 +1,9 @@
+export enum ClassError {
+  noOption = 'Selecciona una opción de la lista',
+}
+
+export enum NumberError {
+  empty = 'Campo Vacío',
```

Comment:
This could be catch in generalRules

Reply:
The problem with using generalRules is that, currently, it only works for string types. This one is being used for a number type value.

Reply:
oh, I c. For time constraints, this is fine, but it could have been a good approach to also implement one for number type